gamestonk_terminal/fred/fred_api.py

```diff
@@ -0,0 +1,122 @@
+from fredapi import Fred
+import argparse
+import matplotlib.pyplot as plt
+from gamestonk_terminal.helper_funcs import (
+    parse_known_args_and_warn,
+)
+from gamestonk_terminal.config_terminal import API_FRED_KEY
+
+
+def get_GDP(l_args):
+    fred = Fred(api_key=API_FRED_KEY)
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="GDP",
+        description="""
+                GDP
+            """,
+    )
+    parser.add_argument(
+        "-n",
+        dest="n_to_get",
+        type=int,
+        default=-1,
+        required=False,
+        help="Number of GDP Values to Grab",
+    )
+
+    parser.add_argument(
+        "-s",
+        dest="start_date",
+        type=str,
+        default="1/1/2020",
+        required=False,
+        help="Date to Start",
+    )
+
+    parser.add_argument(
+        "-p", dest="plot_", type=bool, default=False, required=False, help="Plot GDP"
```

Comment:
I'd keep the convention that has being used, without the "_" at the end of the variable name